ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallOptions.kt

```diff
@@ -43,6 +43,26 @@ data class PaywallOptions internal constructor(
         dismissRequest = builder.dismissRequest,
     )
 
+    override fun hashCode(): Int {
+        var result = offeringSelection.offeringIdentifier.hashCode()
+        result = 31 * result + shouldDisplayDismissButton.hashCode()
+        result = 31 * result + mode.hashCode()
```

Comment:
Right now only using the offeringId+shouldDisplayDismissButton+mode as identifier of the paywall. The others are objects with callbacks which would be much trickier to compare.